wasm/examples/node/ttt.js

```diff
@@ -5,29 +5,34 @@
 
 // Referencing the built package for convenience. Replace with require('pure-evm') when
 // using on one's own.
-const pure_evm = require('../../pkg');
-const ethers = require('ethers');
-const assert = require('assert');
+const pure_evm = require("../../pkg-node");
+const ethers = require("ethers");
```

Comment:
Formatting changes are kind of a lost cause without adding a formatter/prettifier to the build. If it's easy to revert (or you want to add a format step), great. Otherwise, idc.